src/utils/utils.js

```diff
@@ -63,7 +63,11 @@ Utils.buildCCSSFn = function(js, attr) {
 };
 
 Utils.buildAndExecuteFn =  function (js) {
-  return Utils.buildCCSSFn(js, ['$zoom'])(10);
+  return Utils.buildCCSSFn(
+    R.map(
```

Comment:
I'm sure this works like a charm, but I'm not able to "process" these 4 lines... Too much magic for me 😓